src/error.php

```diff
@@ -5,26 +5,35 @@
 $error_msg = $_GET['error_msg'] ?? $error_msg;
 
 ?>
-
-<html>
+<!DOCTYPE html>
+<html lang="en">
 <head>
+    <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0">
-    <link rel="stylesheet" href="css/toggle-bootstrap.min.css">
-    <link rel="stylesheet" href="css/toggle-bootstrap-dark.min.css">
-    <link rel="stylesheet" href="css/main.css">
+
+    <link rel="preload" href="<?=$config['upload_access_path']?>js/setTheme.js" as="script">
+
+    <link rel="stylesheet" href="<?=$config['upload_access_path']?>css/toggle-bootstrap.min.css">
+    <link rel="stylesheet" href="<?=$config['upload_access_path']?>css/toggle-bootstrap-dark-overlay.min.css" onload="this.disabled = true; window.__darkCssLoaded = true; if (window.__updateTheme) { window.__updateTheme(); }">
+
+    <link rel="stylesheet" href="<?=$config['upload_access_path']?>css/main.css">
     <title>Error - <?php echo $config['page_title']; ?></title>
     
-    <link rel="apple-touch-icon" sizes="180x180" href="icons/apple-touch-icon.png">
-    <link rel="icon" type="image/png" sizes="32x32" href="icons/favicon-32x32.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="icons/favicon-16x16.png">
-    <link rel="manifest" href="manifest.webmanifest">
-    <link rel="mask-icon" href="icons/safari-pinned-tab.svg" color="#5c5cbc">
-    <link rel="shortcut icon" href="favicon.ico">
+    <link rel="apple-touch-icon" sizes="180x180" href="<?=$config['upload_access_path']?>icons/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="<?=$config['upload_access_path']?>icons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="<?=$config['upload_access_path']?>icons/favicon-16x16.png">
+    <link rel="manifest" href="<?=$config['upload_access_path']?>manifest.webmanifest">
+    <link rel="mask-icon" href="<?=$config['upload_access_path']?>icons/safari-pinned-tab.svg" color="#5c5cbc">
+    <link rel="shortcut icon" href="<?=$config['upload_access_path']?>favicon.ico">
     <meta name="msapplication-TileColor" content="#5c5cbc">
-    <meta name="msapplication-config" content="icons/browserconfig.xml">
+    <meta name="msapplication-config" content="<?=$config['upload_access_path']?>icons/browserconfig.xml">
```

Comment:
Why do you use `upload_access_path` here? That option is just for accessing your uploads relative to `base_url`, so this assumes that the assets are uploaded files. This should work because of the rewriting, but wouldn't it make more sense to have all these load from `base_url`, since that's what that config option is for?

Reply:
If you loaded `https://example.com/404/test`, no styles would load as the paths to the styles were all relative.

Base URL is the path everything is located at, at default `/`. By using these, we can specify an absolute URI to stylesheets and JS, so instead of `css/...` it becomes `/css/...`. If the user has the files at a gallery path, it becomes `/gallery/css/...`.

Reply:
I haven't experienced this bug. The styles should load properly due to the URL rewriting done through `.htaccess` and now `rewrite.php`. That's what the
```
RewriteRule css/(.*) css/$1
RewriteRule js/(.*) js/$1
RewriteRule icons/(.*) icons/$1
```
lines are for.

However, I still don't really understand why this should be `upload_access_path`. `upload_access_path` specifies the path relative to `base_url` for loading uploaded images.

For example, if a user has their site at `https://www.example.com`, they set their `base_url` to `https://www.example.com`. Then if they want to access an upload called `myfile.txt` at `https://www.example.com/myfile.txt`, they would set `upload_access_path` to `/`. In this case, the changes you proposed work fine. 

However, if a user wants to instead access their site at `https://www.example.com/uploader` and they keep `upload_access_path` as `/`, using `upload_access_path` for the assets would attempt to load files like `https://www.example.com/css/main.css`, which doesn't exist since the files are actually at `https://www.example.com/uploader/css/main.css`.

In addition, if a user decides to set their `upload_access_path` to something like `/files/`, then the browser would try to access assets like `https://www.example.com/files/css/main.css`, which also doesn't exist.

The way to fix this is to use `base_url` and the `join_paths` function (since we can't assume that users' config will start and end with `/`), however that shouldn't be necessary since the paths should be rewritten automatically.

Can you try cloning the latest version of the repo and changing the links back to relative ones to see if this bug still occurs since it doesn't for me?

Reply:
I misunderstood the config options -- my bad!

Hmm that's odd about the rewrites. Maybe something to do with my Apache install...? I'll test it.